src/CareTogether.Api/Controllers/ConfigurationController.cs

```diff
@@ -1,70 +1,81 @@
+            IAuthorizationEngine authorizationEngine
+        )
         {
             //TODO: Delegate this controller's methods to a manager service
-            this.policiesResource = policiesResource;
-            this.featureManager = featureManager;
-            this.authorizationEngine = authorizationEngine;
+            _PoliciesResource = policiesResource;
+            _FeatureManager = featureManager;
+            _AuthorizationEngine = authorizationEngine;
         }
 
-
         [HttpGet("/api/{organizationId:guid}/[controller]")]
         public async Task<ActionResult<OrganizationConfiguration>> GetOrganizationConfiguration(Guid organizationId)
         {
-            var result = await policiesResource.GetConfigurationAsync(organizationId);
+            OrganizationConfiguration? result = await _PoliciesResource.GetConfigurationAsync(organizationId);
             return Ok(result);
         }
 
         [HttpPut("/api/{organizationId:guid}/[controller]/roles/{roleName}")]
-        public async Task<ActionResult<OrganizationConfiguration>> PutRoleDefinition(Guid organizationId,
-            string roleName, [FromBody] RoleDefinition role)
+        public async Task<ActionResult<OrganizationConfiguration>> PutRoleDefinition(
+            Guid organizationId,
+            string roleName,
+            [FromBody] RoleDefinition role
+        )
         {
             if (!User.IsInRole(SystemConstants.ORGANIZATION_ADMINISTRATOR))
+            {
                 return Forbid();
-            var result = await policiesResource.UpsertRoleDefinitionAsync(organizationId, roleName, role);
+            }
+
+            OrganizationConfiguration? result = await _PoliciesResource.UpsertRoleDefinitionAsync(
+                organizationId,
+                roleName,
+                role
+            );
             return Ok(result);
         }
 
         [HttpGet("/api/{organizationId:guid}/{locationId:guid}/[controller]/policy")]
-        public async Task<ActionResult<EffectiveLocationPolicy>> GetEffectiveLocationPolicy(Guid organizationId, Guid locationId)
+        public async Task<ActionResult<EffectiveLocationPolicy>> GetEffectiveLocationPolicy(
+            Guid organizationId,
+            Guid locationId
+        )
         {
-            var result = await policiesResource.GetCurrentPolicy(organizationId, locationId);
+            EffectiveLocationPolicy? result = await _PoliciesResource.GetCurrentPolicy(organizationId, locationId);
             return Ok(result);
         }
 
         [HttpGet("/api/{organizationId:guid}/{locationId:guid}/[controller]/flags")]
-        public async Task<ActionResult<CurrentFeatureFlags>> GetLocationFlags(Guid organizationId)
+        public async Task<ActionResult<CurrentFeatureFlags>> GetLocationFlags()
         {
-            var result = new CurrentFeatureFlags(
-                InviteUser: await featureManager.IsEnabledAsync(nameof(FeatureFlags.InviteUser)),

```

Comment:
This is one instance where the automatic naming changes are not great since we lose the clarity of which feature flag corresponds to which property, but arguably we could be returning these in a dictionary anyways. I'm fine with this change for now, but maybe when merging this PR we should create an issue to change `CurrentFeatureFlags` to be a `Dictionary<string, bool>` where the initializer uses `nameof` to set the key? We could even generate this collection dynamically by iterating through all the values on `FeatureFlags`.

Reply:
I'm not sure why it removed the parameter names, but adding it does not show an error/warning, so I think we are good now :)